pkg/blobstore/grpcservers/byte_stream_server_test.go

```diff
@@ -186,7 +186,7 @@ func TestByteStreamServer(t *testing.T) {
 		).Return(buffer.NewBufferFromError(status.Error(codes.NotFound, "Blob not found")))
 
 		req, err := client.Read(ctx, &bytestream.ReadRequest{
-			ResourceName: "///fedora28//blobs/09f34d28e9c8bb445ec996388968a9e8/////7/",
+			ResourceName: "fedora28/blobs/09f34d28e9c8bb445ec996388968a9e8/7",
```

Comment:
I'm not 100% sure what was the intention of the original test.... let me know if this should be reverted and made work as is.
As you can expect the error changed with the new parsing logic that allows `//` in the instance name.